packages/portal/nuxt.config.js

```diff
@@ -75,6 +75,7 @@ export default {
           origin: [process.env.PORTAL_BASE_URL].concat(process.env.APP_FEEDBACK_CORS_ORIGIN?.split(',')).filter((origin) => !!origin)
         }
       },
+      homeSlug: process.env.APP_HOME_SLUG,
```

Comment:
this isn't used anywhere, would the idea be to use it as an identifier in the homePage contentful query?
Does it need a default value here?

Reply:
Removed to avoid distraction from what this PR is for.